### PR TITLE
Ping360: Add checkbox for heading integration

### LINKED ISSUE #890: Add option to remove ping360 heading integration
From @kklemens:
> Ping-Viewer t2.0.6
Recommendations:
Have an option to turn off the Ping360 chart rotation if someone doesn’t want it. We have some “old school” customers.

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix #890

### CODE REVIEW

File: src/sensor/ping360.cpp [hunk 1]
```diff
@@ -531,4 +526,23 @@ void Ping360::resetSettings()
...
+        disconnect(MavlinkManager::self(), &MavlinkManager::mavlinkMessage, this, &Ping360::processMavlinkMessage);
+        _heading = 0;
+        emit headingChanged();
+    }
+}
+
+void Ping360::processMavlinkMessage(const mavlink_message_t& message)
+{
+    mavlink_attitude_t attitude;
+    mavlink_msg_attitude_decode(&message, &attitude);
```
Comment (jaxxzer) [inline src/sensor/ping360.cpp]: I think we should check the message type of `message` or change the type of the parameter to `const mavlink_attitude_t&`

File: qml/Ping360Visualizer.qml [hunk 2]
```diff
@@ -379,9 +379,22 @@ Item {
                 onCurrentTextChanged: waterfall.theme = currentText
             }
 
+            CheckBox {
+                id: headingCorrection
```
Comment (jaxxzer) [inline qml/Ping360Visualizer.qml]: nitpick: it is funny to have two terms for the same thing: heading correction and heading integration

File: src/sensor/ping360.cpp (hunk 1 above)
Comment (patrickelectric) [reply src/sensor/ping360.cpp]: We only decode attitude messages, but done.

File: qml/Ping360Visualizer.qml (hunk 2 above)
Comment (patrickelectric) [reply qml/Ping360Visualizer.qml]: Done.

**Pull request merged**